AstarteDeviceSDKCSharp/Data/IAstarteFailedMessageStorage.cs

```diff
@@ -38,11 +38,11 @@ public interface IAstarteFailedMessageStorage
 
         AstarteFailedMessageEntry? PeekFirstCache();
 
-        void AckFirst();
+        void AckFirst(AstarteFailedMessageEntry failedMessages);
```

Comment:
What do you think to rename this into `Ack/Reject` since you are `ack/reject` a specified message?
```suggestion
        void Ack(AstarteFailedMessageEntry failedMessages);
```

Reply:
Changed